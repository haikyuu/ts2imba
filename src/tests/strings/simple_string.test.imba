import {build} from "../../index"
import * as imbac from 'imba/compiler'

let tsx-code = '''
"hello"
'''

test 'simple_string' do
	const result = await build tsx-code
	expect(result.code).toMatchSnapshot()
	const out = imbac.compile(result.code, sourceId: 'sth')
	console.log out.js
