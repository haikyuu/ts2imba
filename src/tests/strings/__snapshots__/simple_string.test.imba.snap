// Vitest Snapshot v1

exports[`simple_string 1`] = `
"\\"hello\\"
"
`;
